dist/profile/manifests/datadog_ssl_check.pp

```diff
@@ -0,0 +1,29 @@
+#
+# Define datadog check for ssl expiration
+# Inspired by: https://workshop.avatarnewyork.com/project/datadog-ssl-expires-check/
+#
+
+class profile::datadog_ssl_check (
+  $sites = undef,
+){
+
+  include datadog_agent
+
+  file { 'ssl_check_expire_days.py':
+    ensure => present,
+    source => "puppet:///modules/${module_name}/datadog_ssl_check/ssl_check_expire_days.py",
+    path   => '/etc/dd-agent/checks.d/ssl_check_expire_days.py',
```

Comment:
I believe this should have a `require` that the `dd-agent` package has been installed to make sure this directory will be created before this File resource executes.

It might be enough to `require => Class[datadog_agent]`, but I'm not sure off the top of my head